### ISSUE TITLE
sos metadata: some characters are reserved

### ISSUE THREAD

**Issue by greut** [exoscale/cli#39]
```console
> exo.exe sos metadata add yoanyoanyoan a%20b.zip "a&amp;b" "c"
Put https://ppsos-ch-dk-2.exo.io/yoanyoanyoan/a%2520b.zip: net/http: invalid header field name "x-amz-meta-a&amp;b"
```

**Comment by pierre-emmanuelJ** [comment]
So this error is explicite, by putting this meta `"x-amz-meta-a&amp;b"` you can deduct that value is invalid

**Comment by greut** [comment]
```console
> exo.exe sos metadata add yoanyoanyoan a%20b.zip "a%20b" "c"
The request signature we calculated does not match the signature you provided. Check your key and signing method.
```

**Comment by pierre-emmanuelJ** [comment]
yes sorry my bad!

**Comment by pierre-emmanuelJ** [comment]
It's working with `&, >, "` for example when key is added to the header it's encoded
BUT with `%` it's not encoded like the other characters

**Comment by falzm** [comment]
Fixed in #328.
